sequelize/seeders/04-demo-player.js

```diff
@@ -1,45 +1,18 @@
+const faker = require('faker');
+
 module.exports = {
     up: async function (queryInterface, Sequelize) {
-        return queryInterface.bulkInsert('players', [{
-            firstName: 'Evgeny',
-            lastName: 'Butkevich'
-        },
-        {
-            firstName: 'Anton',
-            lastName: 'Fomenok'
-        },
-        {
-            firstName: 'Mikhail',
-            lastName: 'Sudnikovich'
-        },
-        {
-            firstName: 'Ilya',
-            lastName: 'Faley'
-        },
-        {
-            firstName: 'Arthur',
-            lastName: 'Shket'
-        },
-        {
-            firstName: 'Evgeny',
-            lastName: 'Karlenok'
-        },
-        {
-            firstName: 'Anton',
-            lastName: 'Kucher'
-        },
-        {
-            firstName: 'Ilya',
-            lastName: 'Sinko'
-        },
-        {
-            firstName: 'Evgeny',
-            lastName: 'Gorokhovich'
-        },
-        {
-            firstName: 'Vova',
-            lastName: 'Linnik'
-        }]);
+        const recordsList = [];
+
+        for (let i = 1; i <= 50; i++) {
+            recordsList.push({
+                firstName: faker.name.firstName(0),
```

Comment:
Why do you need to call (0)?

Reply:
(method) firstName(gender?: number): string
(method) firstName(gender?: number): string
0 - male
1 - female